youtube_dl/extractor/rbmaradio.py

```diff
@@ -7,22 +7,20 @@
 from .common import InfoExtractor
 from ..utils import (
     ExtractorError,
+    clean_html
 )
 
 
 class RBMARadioIE(InfoExtractor):
-    _VALID_URL = r'https?://(?:www\.)?rbmaradio\.com/shows/(?P<videoID>[^/]+)$'
+    _VALID_URL = r'https?://(?:www\.)?rbmaradio\.com/shows/.*/episodes/(?P<videoID>[^/]+)$'
```

Comment:
This allows URLs like https://www.rbmaradio.com/shows//episodes/ford-lopatin-live-at-primavera-sound-2011` that is a priori invalid.